### DISCUSSION TITLE
Feature request: make the sponsorer achievement hidable

### DISCUSSION THREAD

**Discussion by ankitsxchdeva** [community/community#9459] [Other Feature Feedback, Questions, & Ideas]
Hey crew! Not sure if this is the right place to put this but I was wondering if would be possible to make the sponsoring achievement togglable in `Settings/Profile/Profile settings`. Similar to how the Arctic Code Vault badge and Pro badges are.

I'm not sure if others would utilize this but I know I definitely would. Thank you for everything that you do!

**Comment by cyntler** [comment]
@ankitsxchdeva Yes, IMO each one achievement should have the option to hide it from the profile.

**Comment by jlord** [comment]
Hiya! This is great feedback, thank you. It makes sense that they should all have the same visibility options. I'll open an issue on our end with the team 👍
